lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -123,7 +125,7 @@ var queryTemplate = Node.template([
     '_data AS ( ',
     '  SELECT id AS __obs_id__, {{=it.obsColumns}}',
     '  FROM cdb_dataservices_client._OBS_GetData_exception_safe(',
-    '    (SELECT ARRAY_AGG((the_geom, cartodb_id)::geomval) FROM _source),',
+    '    (SELECT geomvals FROM _summary),',
```

Comment:
Ok to avoid problems, I'll move the geomvals back its place and add the where to it.